main.go

```diff
@@ -78,17 +78,11 @@ func uploadApplications(configs Configs, service *androidpublisher.Service, appE
 func updateTracks(configs Configs, service *androidpublisher.Service, appEdit *androidpublisher.AppEdit, versionCodes []int64) error {
 	editsTracksService := androidpublisher.NewEditsTracksService(service)
 
-	allTracks, err := getAllTracks(configs.PackageName, service, appEdit)
+	_, err := getAllTracks(configs.PackageName, service, appEdit)
```

Comment:
why do we need this call? just for logging the available tracks?

Reply:
This call seems unnecessary, the return value is unused and we should not build on side effects which is just logging.

Reply:
Yes it was used only for logging, which seemed at least somewhat useful.

Reply:
ok, let's merge it